src/index.html

```diff
@@ -9,6 +9,93 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="nav__div">
+        
+        <img
+         class="image"
+         href="home"
+         src="./images/logo.png"
+         alt="moyo"
+        >
+      </div>
+      
+      <nav class="nav__lint">
```

Comment:
probably a typo, and you wanted to name it `list`
but it's not a `list` actually. it's directly `nav`

```suggestion
      <nav class="nav">
```